example.tf

```diff
@@ -12,6 +12,22 @@ resource "rackcorp_server" "example" {
   memory_gb        = 4
   root_password    = "a-secret-password"
 
+  /*
+  firewall_policies = [
+    {
+      direction       = "INBOUND"
+      policy          = "ALLOW"
+      protocol        = "TCP"
+      port_to         = "80"
+      port_from       = "80"
+      ip_address_from = "203.0.113.5"
```

Comment:
This is not a range, which is implied (I think) This is source IP and destination IP.

Same with port.